bin/droid-mysql

```diff
@@ -17,7 +18,10 @@ require $loader;
 
 $application = new Application('Droid Mysql', '1.0.0');
 $application->setCatchExceptions(true);
-$application->add(new Droid\Plugin\Mysql\Command\MysqlDumpCommand());
-$application->add(new Droid\Plugin\Mysql\Command\MysqlDumpAllCommand());
-$application->add(new Droid\Plugin\Mysql\Command\MysqlLoadCommand());
+
+$registry = new DroidPlugin($application);
+foreach ($registry->getCommands() as $command) {
+    $application->add($command);
+}
+
 $application->run();
```

Comment:
Does anyone know of any reason not to use `DroidPlugin` in the `bin/droid-*` command runners?  It is used in this case to avoid repetition of the command construction logic.